code/_helpers/areas.dm

```diff
@@ -93,7 +92,7 @@
 	. = !istype(A,/area/space)
 
 /proc/is_not_shuttle_area(var/area/A)
-	. = !istype(A,/area/shuttle)
+	. = !(istype(A) && (A.area_flags & AREA_FLAG_SHUTTLE))
```

Comment:
is `!(istype(A)` right here? Wouldn't this always fail on areas, flag or not (since areas are usually of type area)

Reply:
Discussed on Discord, rewrote the logic to be clearer.